components/FormCreate.js

```diff
@@ -272,13 +273,14 @@ export default function FormCreate() {
                 </svg>
               </PlusButton>
             </ButtonWrapper>
+            {isLoading ? <ImageSkeleton/> :
             <ImageContainer>
               <SearchImage
-                src={imageSearch.hits[increment].largeImageURL}
+                src={imageSearch.hits[increment].webformatURL}
                 fill
                 alt="Pixabay Image"
               />
-            </ImageContainer>
+            </ImageContainer>}
           </ContainerSwitchesAndPicture>
         ) : (
           ""
```

Comment:
it seems that you don't want to display anything if `typingInSearchBar` is false. you can then switch from the ternary operator to a shorter version, the `&&`. This returns the statement after it if the previous is `true` and nothing otherwise: 

```js
typingInSearchbar &&  (
...
)
```
You can omit then the ` : ""` here.

Reply:
Strangely when i changed to the && operator the app started showing a 0 as text instead of the image when the fetched array was empty, e.g. when we searched "Paragli".

After switching back to a ternary operator and returning "null" the 0 disappeared. So currently i would resort to the ternary operator unless you could clear this issue up, while also using the &&.